advanced/Templates/gravity.db.sql

```diff
@@ -1,3 +1,12 @@
+PRAGMA FOREIGN_KEYS=ON;
+
+CREATE TABLE "group"
+(
+	id INTEGER PRIMARY KEY AUTOINCREMENT,
+	enabled BOOLEAN NOT NULL DEFAULT 1,
```

Comment:
We can use `TRUE` and `FALSE` instead of integers for booleans.

Reply:
> SQLite does not have a separate Boolean storage class. Instead, Boolean values are stored as integers 0 (false) and 1 (true).

https://www.sqlite.org/datatype3.html

I am experiencing issues with using `TRUE` and `FALSE` on older versions of `sqlite3`. While replacing `1` with `TRUE` works with version 3.26.0, it does **not work** with version 3.16.2 (Jan 2017) as shipped, e.g., by recent Armbian Linux.

Reply:
Update: I did some further research on this. SQLite supports true/false literals only since [3.23.0](https://www.sqlite.org/releaselog/3_23_0.html):

> Recognize `TRUE` and `FALSE` as constants. (For compatibility, if there exist columns named "true" or "false", then the identifiers refer to the columns rather than Boolean constants.)

Reply:
This is even more severe as I though initially, when we changed
```sql
enabled BOOLEAN NOT NULL DEFAULT 1,
```
to
```sql
enabled BOOLEAN NOT NULL DEFAULT TRUE,
```
this actually changed the type to `TEXT` and stored the string causing unexpected behavior: As `FALSE` is **not** an alias for `0` in older versions, it is actually `TRUE`.